.github/workflows/continous-integration-workflow.yml

```diff
@@ -0,0 +1,28 @@
+name: Continuous Integration
+on: [push, pull_request]                      # Will make the workflow run every time you push and riase PR to any branch
+
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest      # Sets the build environment a machine with the latest Ubuntu installed
+    steps:
+    - name: Checkout the code    # Adds a step to checkout the repository code
+      uses: actions/checkout@v2 
+    - name: Build the application
+      run: |
+        docker build --target development --tag todo-app:dev .
+    - name: Run the unit and integration tests
+      working-directory: .
+      run: | 
+        docker build --target test --tag todo-app:test .
+        docker run --env-file .env.test --mount type=bind,source="$(pwd)",target=/app/ todo-app:test todo_app_tests
```

Comment:
Is there any reason to include a mount here? I don't see it causes any issue, but the test container should already have all the relevant files in